lisa/trace.py

```diff
@@ -230,61 +261,50 @@ class TraceView(Loggable, TraceBase):
         mimics a regular :class:`Trace` using :func:`getattr`.
     """
 
-    def __init__(self, trace, window):
+    def __init__(self, trace, window, clear_base_cache=False):
         super().__init__()
-
         self.base_trace = trace
 
-        t_min = window[0]
-        t_max = window[1]
+        # evict all the non-raw dataframes from the base cache, as they are
+        # unlikely to be used anymore.
+        if clear_base_cache:
+            self.base_trace._cache.clear_all_events(raw=False)
```

Comment:
I was unsure either. For RTATestBundle.trace, it's pretty obvious what to do but otherwise it depends on what memory you are ready to spend. This depends on the trace, the memory available, and the share of that available memory that you are ready to spend for that LISA process.